mobile/README.md

```diff
@@ -0,0 +1,15 @@
+# GnoSocial Mobile APP
+
+## Requirements
+
+- Expo Requiments: https://docs.expo.dev/get-started/installation/
+
+## 🚀 How to use
+
+```sh
+# to run on ios:
+yarn ios 
+
+# to run on android:
+yarn android 
```

Comment:
Same as for ios above. This prints "expo: command not found". But if I first enter `yarn` then this builds and runs.